exec/java-exec/src/test/java/org/apache/drill/exec/expr/fn/impl/TestSqlPatterns.java

```diff
@@ -446,6 +446,61 @@ public void testSqlPatternComplex() {
     assertEquals(1, sqlPatternComplex.match(0, byteBuffer.limit(), drillBuf)); // should match
   }
 
+  @Test
+  public void testSqlPatternContainsMUltipleMatchers() {
```

Comment:
These tests are incomplete. We do not test:

* Pattern length = input length, match
* Pattern length = input length, no match
* Pattern length < input length
* Pattern length = 0
* Non-ASCII paterns and/or data
* Data longer than the internal table sizes
* Patterns of each of the special lengths for each of the above cases: match, not match, shorter, equals, longer

Reply:
If tests exist, and you have verified that each of the lengths is, in fact, tested by those tests, then I'm fine with adding a comment identifying the cases and where to find the existing tests. The point is to make sure all variations are fully tested -- one way or another.